tests/terraform_compliance/steps/its_key_value_test_cases/root_and_values_2.json

```diff
@@ -0,0 +1,13 @@
+{
+    "stash": [
+        {
+            "id": "a",
+            "foo": 2000,
+            "values": {
+                "foo": 1000
+            }
+        }
+    ],
+    "args": ["foo", 1000],
+    "expected_ids": ["a"]
```

Comment:
Items inside `values` take precedence. Is that right?

Reply:
True, but couldn't get the point about expected_ids ?

Reply:
> True, but couldn't get the point about expected_ids ?

That wasn't very clear, sorry. 
I was trying to indicate that you needed to take both `root_and_values` tests into consideration to demonstrate the precedence.